googleCalender.js

```diff
@@ -0,0 +1,427 @@
+const max = 50;
+async function usersFullSync({ dataStore, client }) {
+  const responseData = await client.fetch(
+    "admin/directory/v1/users?customer=my_customer&projection=full&viewType=admin_view"
+  );
+
+  if (!responseData.ok) {
+    throw new Error(
+      `Events sync failed ${responseData.status}:${responseData.statusText}.`
+    );
+  }
+  const data = await responseData.json();
+
+  for (let user of data.users) {
+    let userData = {
+      agreed_To_Terms: user.agreedToTerms,
+      archived: user.archived,
+      change_Password_At_Next_Login: user.changePasswordAtNextLogin,
+      cocustomer_Id: user.customerId,
+      etag: user.etag,
+      id: user.id,
+      include_In_Global_Address_List: user.includeInGlobalAddressList,
+      is_Admin: user.isAdmin,
+      is_Enforced_In_2_Sv: user.isEnforcedIn2Sv,
+      is_Mailbox_Setup: user.isMailboxSetup,
+      kind: user.kind,
+      name_Family_Name: user.name.familyName,
+      name_Full_Name: user.name.fullName,
+      nname_Given_Name: user.name.givenName,
+      org_Unit_Path: user.orgUnitPath,
+      primary_Email: user.primaryEmail,
+      recovery_Email: user.recoveryEmail,
+      suspended: user.suspended,
+    };
+
+    dataStore.save("users", userData);
+
+    await Events(dataStore, client, user.primaryEmail);
+  }
+}
+
+async function Events(dataStore, client, primaryEmail) {
+  const responseDataEvent = await client.fetch(
+    `calendar/v3/calendars/${primaryEmail}/events?maxResults=${max}&showDeleted=false&singleEvents=true&orderBy=startTime`
+  );
+
+  if (!responseDataEvent.ok) {
+    throw new Error(
+      `Events sync failed ${responseDataEvent.status}:${responseDataEvent.statusText}.`
+    );
+  }
+
+  let dataEvent = await responseDataEvent.json();
```

Comment:
Refer to previous comment
```suggestion
  const dataEvent = await responseDataEvent.json();
```